src/Recommendations/Rule/RelationsOrder.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace eLife\Recommendations\Rule;
+
+use DomainException;
+use eLife\ApiSdk\Model\Article;
+use eLife\Recommendations\RuleModel;
+
+class RelationsOrder
+{
+    private $typeOrder = [
+        'retraction',
+        'correction',
+        'external-article',
+        'registered-report',
+        'replication-study',
+        'research-advance',
+        'scientific-correspondence',
+        'research-article',
+        'tools-resources',
+        'feature',
+        'insight',
+        'editorial',
+        'short-report',
+        'collection',
+        'podcast-episode',
+        'podcast-episode-chapter',
+    ];
+
+    public function filter($articles)
```

Comment:
This confused me initially as it's not filtering.

A bigger change would be to make it `__invoke()` and pass it as a `callable` to `NormalizedPersistence`, which then uses it to do the `sort()`. But let's not worry about that now.